style.css

```
@@ -0,0 +1,123 @@
+.menu{
+    position: fixed;
+    inset-block-start: 0;
+    inset-block-end: 75%;
+    inset-inline: 0;
+    display: flex;
+    align-items: center;
+    padding-inline: 2rem;
+    padding-block: 0rem;
+    justify-content: space-between;
+    background-color: #ffc72cc4;
+    z-index: 1;
+    margin-inline: 0;
+    
+}
+.logo-chicken{
+    width: 10rem;
+    height: 10rem;
+}
+a{
+    text-decoration: none;
+    color: rgb(97, 10, 10);
+    margin-inline: 1rem ;
+    position: relative;
+    transition: transform 1s ease, text-decoration 1.5s ease;
+}
+a:hover{
+    text-decoration: underline;
+    transform: scale(1.3)
+} 
+.buttons{
+    display: flex;
+    align-items: flex-end;
+    font-family: 'Sedgwick Ave Display';
```

Ai putea pune și font-uri de fallback

```suggestion
    font-family: 'Sedgwick Ave Display', cursive;
```